Plugins/Player/Player.cpp

```diff
@@ -1620,3 +1608,90 @@ NWNX_EXPORT ArgumentStack CloseStore(ArgumentStack&& args)
 
     return {};
 }
+
+NWNX_EXPORT ArgumentStack SetTlkOverride(ArgumentStack&& args)
+{
+    static Hooks::Hook s_SendServerToPlayerSetTlkOverrideListHook =
```

Comment:
I think it might be better to just do it like SetCustomToken - i.e. override for the current session only, and the upper layer can keep a list of overrides in sqlite and apply them OnClientEnter.

Reply:
Works for me